src/views/splash/features/features-banner.jsx

```diff
@@ -0,0 +1,41 @@
+const FormattedMessage = require('react-intl').FormattedMessage;
+const React = require('react');
+
+const FlexRow = require('../../../components/flex-row/flex-row.jsx');
+const TitleBanner = require('../../../components/title-banner/title-banner.jsx');
+
+const thumbnail = require('./high-contrast-thumbnail.png');
+
+require('./features-banner.scss');
+
+const FORUM_URL = 'https://scratch.mit.edu/discuss/topic/683877/';
```

Comment:
This is not the correct link. Still waiting on the forum post to be created.

Reply:
This has been updated with the correct link. The post isn't live yet, but the url should be the same when it does go live.